runtime-modules/argo-bridge/src/tests/mod.rs

```diff
@@ -316,6 +316,109 @@ fn finalize_inbound_transfer_with_insufficient_bridge_mint() {
     });
 }
 
+#[test]
+fn revert_outbound_transfer_success() {
+    with_test_externalities_custom_mint_allowance(joy!(1000), || {
+        let remote_chains = BoundedVec::try_from(vec![1u32]).unwrap();
+        let parameters = BridgeConstraints {
+            operator_account: Some(account!(1)),
+            pauser_accounts: None,
+            bridging_fee: None,
+            thawn_duration: None,
+            remote_chains: Some(remote_chains),
+        };
+        ArgoBridge::update_bridge_constrains(RuntimeOrigin::root(), parameters).unwrap();
+
+        let transfer_id = 1u64;
+        let revert_amount = joy!(123);
+        let revert_account = account!(2);
+        let rationale = "test".as_bytes().to_vec();
+        let result = ArgoBridge::revert_outbound_transfer(
+            RuntimeOrigin::signed(account!(1)),
+            transfer_id,
+            revert_account,
+            revert_amount,
+            rationale.clone(),
+        );
+        assert_ok!(result);
+        assert_eq!(Balances::free_balance(revert_account), revert_amount);
+        last_event_eq!(RawEvent::OutboundTransferReverted(
+            transfer_id,
+            revert_account,
+            revert_amount,
+            rationale,
+        ));
+    });
+}
+
```

Comment:
Despite my previous comment I will point this out :
transfer revert failing test case for bridge status not active is missing

Reply:
Yes, we should add this as well. Let me do this in next PR though, at this branch there are small issues with bridge status that I fixed on my next branch